andela/AndelaEatsUI
# #161079529 Implement admin viewing engagements (UI)

#### What does this PR do?
Add view page for an admin to see all engagements

#### Description of Task to be completed?
- add engagement icon and link to the sidebar
- add engagement react component
- write test suites for engagement component

#### How should this be manually tested?
- Log in to the app
- Switch to the admin dashboard from the user profile drop-down menu or [here](http://andelaeats-dev.andela.com:3000/admin/engagements)
- You should see a static page displaying meals

#### What are the relevant pivotal tracker stories?
- [161079529](https://www.pivotaltracker.com/n/projects/2178749/stories/161079529)

#### Background Context
Nil

#### Screenshots (If Applicable)
![screen shot 2018-11-14 at 9 50 49 am](https://user-images.githubusercontent.com/21138774/48470838-d8c9b600-e7f2-11e8-9a4a-557233e363e6.png)
